src/carts/postCartEntry.js

```diff
@@ -71,7 +71,9 @@ function postCartEntry(args) {
     switch (actionState) {
         case ActionStateEnum.NEW_EMPTY_CART:
             return cart.create().then(function (result) {
-                return cart.byId(result.response.body).get();
+                let id = result.response.body;
+                let headers = {'Location': `carts/${id}`};
```

Comment:
@LSantha Have you checked, that if you overwrite headers like this, you still return all required headers properly?